src/libraries/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -29,10 +29,10 @@ public partial class EnumerableQuery<T> : System.Linq.EnumerableQuery, System.Co
         System.Collections.Generic.IEnumerator<T> System.Collections.Generic.IEnumerable<T>.GetEnumerator() { throw null; }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         System.Linq.IQueryable System.Linq.IQueryProvider.CreateQuery(System.Linq.Expressions.Expression expression) { throw null; }
-        System.Linq.IQueryable<S> System.Linq.IQueryProvider.CreateQuery<S>(System.Linq.Expressions.Expression expression) { throw null; }
+        System.Linq.IQueryable<TElement> System.Linq.IQueryProvider.CreateQuery<TElement>(System.Linq.Expressions.Expression expression) { throw null; }
         object System.Linq.IQueryProvider.Execute(System.Linq.Expressions.Expression expression) { throw null; }
-        S System.Linq.IQueryProvider.Execute<S>(System.Linq.Expressions.Expression expression) { throw null; }
-        public override string ToString() { throw null; }
+        TElement System.Linq.IQueryProvider.Execute<TElement>(System.Linq.Expressions.Expression expression) { throw null; }
+        public override string? ToString() { throw null; }
     }
     public static partial class Queryable
     {
```

Comment:
Are all API's implementation was not exist within the project or? I think ref assemblies missing some annotation

Reply:
I didn't see any types requiring nullable annotations leaking out in the public signatures.

Reply:
@stephentoub Would changing type variable names be considered a breaking change here? If that's the case then I should maybe rename `TElement` to `S` in the implementation.

Reply:
> Would changing type variable names be considered a breaking change here?

Technically, yes, in that it could impact some reflection-based scenarios.

However, the chance of this breaking anyone is very low.  This is an explicit interface implementation, so the only way someone would be impacted by this is if they were using reflection.  Further, for this `S` to make a difference, they would need to be reflecting over the reference assembly rather than over the implementation assembly, which is unlikely to be done by anything where such a difference would be meaningful.  And, if the difference is actually meaningful, it's worth calling out that neither S nor TElement actually matches what's in the interface, which is TResult:
https://github.com/dotnet/runtime/blob/14db363c4acf2f5ce3ed81119af3ba81454c80ae/src/libraries/System.Linq.Expressions/ref/System.Linq.Expressions.cs#L252
:smile:

cc: @cston